images/buildkit/config/main.tf

```diff
@@ -0,0 +1,37 @@
+variable "rootless" {
+  description = "Whether to run as root or not."
+  type        = bool
+  default     = true
+}
+
+variable "extra_packages" {
+  description = "Additional packages to install."
+  type        = list(string)
+  default     = []
+}
+
+module "rootfull" {
+  source = "../../../tflib/accts"
+  name   = "root"
+  uid    = 0
+  gid    = 0
+  run-as = 0
```

Comment:
perhaps instead of instantiating accts twice and optionally using one we can have the ternary on `run-as = rootless ? 65532 : 0`?

Reply:
This is actually used (left over from something I was trying before). Removed!